lib/backend/k8s/conversion/conversion.go

```diff
@@ -218,10 +211,10 @@ func (c Converter) PodToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      wepName,
 				Namespace: pod.ObjectMeta.Namespace,
-				Labels:    labels,
+				Labels:    pod.ObjectMeta.Labels,
```

Comment:
nit, but just `pod.Labels` seems to be what we do elsewhere.

Reply:
yup - was fixing that up where I noticed - guess this one slipped by